similarity-data-rest/src/main/java/org/phenotips/data/similarity/rest/internal/DefaultPatientMatchResource.java

```diff
@@ -131,11 +134,17 @@ private Response generateMatchesResponse(
     {
         final List<PatientSimilarityView> matches = this.similarPatientsFinder.findSimilarPatients(patient);
 
-        this.matchStorageManager.saveLocalMatches(matches, patient.getId());
+        Map<PatientSimilarityView, PatientMatch> savedMatches =
+            this.matchStorageManager.saveLocalMatches(matches, patient.getId());
 
-        final MatchedPatientClusterView cluster = new DefaultMatchedPatientClusterView(patient, matches);
+        @SuppressWarnings({ "rawtypes", "unchecked" })
```

Comment:
I think you don't need to suppress any warnings if you do `new HashMap<>()`; on the next line